src/ui/layout/mct-tree.vue

```diff
@@ -430,28 +430,36 @@ export default {
 
             return scrollTopAmount >= treeStart && scrollTopAmount < treeEnd;
         },
+        getLowercaseObjectName(domainObject) {
+            let objectName;
+            if (!domainObject) {
+                return objectName;
+            }
+
+            if (domainObject.name) {
+                objectName = domainObject.name.toLowerCase();
+            }
+
+            if (domainObject.object && domainObject.object.name) {
```

Comment:
this looks like weird case, when does this happen?

Reply:
There's a check for this in the older code as well, I just copied it over.